Lombiq.HelpfulLibraries.Tests/UnitTests/Models/TypedRouteTests.cs

```diff
@@ -184,4 +165,17 @@ void AddArgumentsTest(int addDays, string expect)
 
         return tests;
     }
+
+    [SuppressMessage("Performance", "CA1819:Properties should not return arrays", Justification = "This is just an input class for a unit test.")]
+    public class TypedRouteShouldWorkCorrectlyInput(
+        string expected,
+        Expression<Action<RouteTestController>> actionExpression,
+        (string Name, object Value)[] additional,
+        string tenantName)
+    {
+        public string Expected { get; set; } = expected;
+        public Expression<Action<RouteTestController>> ActionExpression { get; set; } = actionExpression;
+        public (string Name, object Value)[] Additional { get; set; } = additional;
+        public string TenantName { get; set; } = tenantName;
```

Comment:
This suppression is not necessary. The `(string Name, object Value)[] Additional` property doesn't have to be array, because it's only used once in `TypedRoute.CreateFromExpression()` ([here](https://github.com/Lombiq/Helpful-Libraries/pull/259/files#diff-5675bbae9da4a1e0b90627b4121be1fbae5a994085ef6158aa9d4dc2230a98d7R30-R33)) and that expects `IEnumerable<(string Key, object Value)>` value for its `additionalArguments` parameter. So you can make the property `IEnumerable<(string Key, object Value)>` as well.